android/src/main/java/com/aerobotics/DjiMobile/DJIMobile.java

```diff
@@ -127,11 +127,14 @@ public void onRegister(DJIError djiError) {
       @Override
       public void onProductDisconnect() {
         product = null;
+        // TODO: Do we need to stop this?
```

Comment:
@MikeWoots this is the only question I have. Is this necessary? What does this actually achieve?